### PR TITLE
preserve meta when converting block to molecule

### PR DESCRIPTION
**Pull request by fgrunewald**
Fix to the function to_molecule to preserve the meta data of the block. Fixes issue #244.

### CODE REVIEW

**Comment by fgrunewald** [comment]
@pckroon seems like the doc building fails. Locally the doc is built without problems including '-EnW'. Can you try to build it and see, if there is an issue for you.

**Comment by pckroon** [comment]
You may need to add a -A (or -a?) flag locally to make sure you build all the docs, and not just the files that changed.

File: vermouth/tests/test_molecule.py [hunk 1]
```diff
@@ -1243,3 +1244,29 @@ def test_str_method(mol, moltype):
...
             assert '{} {}'.format(0, itype) not in found
+
+def test_to_molecule():
+    """
+    Test if the to molecule function gives
+    expected results.
+    """
+    force_field = vermouth.forcefield.ForceField("test")
+    test_block = vermouth.molecule.Block()
+    test_block.add_edges_from([('A','B'), ('B','C')])
```
Comment (pckroon) [inline vermouth/tests/test_molecule.py:1255]: Missing spaces

File: vermouth/tests/test_molecule.py [hunk 2]
```diff
@@ -1243,3 +1244,29 @@ def test_str_method(mol, moltype):
...
+                            meta={'b': 1}),]
+
+    molecule = test_block.to_molecule()
+
+    ref_bonds = [ Interaction(atoms=(0, 1),
+                            parameters=['a', '0.2', '200'],
+                            meta={'a': 0}),
+                Interaction(atoms=(1, 2),
+                            parameters=['a', '0.1', '300'],
+                            meta={'b': 1}),]
```
Comment (pckroon) [inline vermouth/tests/test_molecule.py:1271]: Pep8 with indentation

**Comment by pckroon** [comment]
Still some tiny pep8 issues, but if you don't feel like fixing them it's fine as-is

**Pull request merged**